packages/builder/src/components/userInterface/temporaryPanelStructure.js

```diff
@@ -327,6 +327,12 @@ export default {
                 key: "datasource",
                 control: TableViewSelect,
               },
+              {
+                label: "Table URL",
+                key: "detailUrl",
+                placeholder: "tableName",
+                control: Input,
```

Comment:
We actually have the `ScreenSelect` control made specifically for selecting screens to link to - that's what's used for choosing the target of a `Link` or a `Card` for example. Might be a good idea to use that here to make it easier for people to pick the right URL.